addon/components/webrtc-troubleshooter/template.hbs

```diff
@@ -3,25 +3,27 @@
     <div class="entry-row time">{{t "webrtcTroubleshoot.troubleshootTimeWarning"}}</div>
   {{/if}}
   {{#if audio}}
-    <div class="entry-row">
-      <div class="entry-label">
-        <i class="fa fa-unlock"></i>
-      </div>
-      <div class="entry-values">
-        {{#if checkingMicPermissions}}
-          {{t "webrtcTroubleshoot.checkMicPermissions"}} <i class="status-icon pc-moon pc-loading pc-spin"></i>
-        {{else if micPermissionsSuccess}}
-          {{t "webrtcTroubleshoot.micPermissionsSuccess"}} <i class="status-icon pc pc-check"></i>
-        {{else}}
-          {{t "webrtcTroubleshoot.micPermissionsFail"}} <i class="status-icon ion-alert-circled"></i>
-          {{#if noMicrophone}}
-            <div class="entry-value">{{t "webrtcTroubleshoot.noMicrophoneForCheck"}}</div>
+    {{#unless skipPermissionsCheck}}
```

Comment:
I think we'd still want them to show, but just show as passed, no?

Reply:
I don't really know what that buys us. I mean they'd always pass right? It just seemed pointless.

Reply:
Ehh, Keeping the information consistent for users is valuable. The diagnostics here emphasizes "you have permissions working" even if that's implicit because of the fact that they're using the desktop app.

Reply:
I'll make the change. It will be dependent on a release of this though: https://github.com/MyPureCloud/webrtc-troubleshooter/pull/51/files